### ISSUE TITLE
README claims no bugs

### ISSUE THREAD

**Issue by ghost** [jacob1/The-Powder-Toy#20]
`README` contains the lines
```
------------------------------------------------------------------------------
-------------------------------------Bugs-------------------------------------
------------------------------------------------------------------------------
None (at least that I know of)
```

But this "Issues" section begs to differ :)

**Comment by jacob1** [comment]
It's not a bug, it's a feature. Who wouldn't want invisible textboxes?

**Comment by ghost** [comment]
@jacob1 What about the lines, wind and backspace?

**Comment by jacob1** [comment]
Fine, invisible textbox feature removed. I also removed these features:
wind+line tool automatically created an undo point on mouseup
ctrl+backspace would automatically select some text for you in certain textboxes
when drawing lines with flat brushes, it would automatically create some extra particles for you

**Comment by ghost** [comment]
I was going to say something about how this could be legitimately closed now that you'd fixed the others, but #16 still remained at the time :)

Of course, there is still one issue remaining...

**Comment by jacob1** [comment]
If you mean the stamp browser thing, it's technically just an implementation detail I don't like and want to change. Not a bug.

(And I increased the limit to 5000 anyway)
